tests/e2e-instrumentation/instrumentation-apache-multicontainer/chainsaw-test.yaml

```diff
@@ -222,8 +219,8 @@ spec:
             ($error == null): true
       - assert:
           resource:
-            ($metrics[?as_string(metric."__name__") == 'otelcol_exporter_sent_spans_total'].value | [0] > `1`): true
-            ($metrics[?as_string(metric."__name__") == 'otelcol_receiver_accepted_spans_total'].value | [0] > `1`): true
```

Comment:
Shouldn't this be at least 2 given that it's the second test we're running against this collector?

Reply:
A new pod is created for the second test as we apply a changed CR which removes the instrumentation annotation for the second container.

Reply:
Ah, I missed that we're running the collector as a sidecar in this test. Everything's fine then. :+1: